pretrain/tinyllama.py

```diff
@@ -167,6 +168,7 @@ def train(fabric, state, train_dataloader, val_dataloader, resume):
         )
 
     running_loss = RunningMean(window=gradient_accumulation_steps, sync_on_compute=False).to(fabric.device)
+    fabric.barrier()
```

Comment:
To ensure we start measuring time only when all processes are ready to enter the loop.

Reply:
Smart